src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchFailuresSpec.groovy

```diff
@@ -120,7 +119,6 @@ class SwitchFailuresSpec extends HealthCheckSpecification {
 
         cleanup:
         lockKeeper.cleanupTrafficShaperRules(swPair.dst.regions)
-        islHelper.restoreIsl(islToBreak)
         database.resetCosts(topology.isls)
```

Comment:
is it reduntant?

Reply:
It was needed because antiflap.portDown() didn't have this cleanup action. I've added the one after your comment.